hotsos/plugin_extensions/openvswitch/event_checks.py

```diff
@@ -93,73 +151,45 @@ def __call__(self, event):
         this case it is possible to have 100% packet drops on the interface
         if that VR has never been a vrrp MASTER. For this scenario we filter
         interfaces whose name matches e.g. qg-3ca935f4-07.
+
+        @param event: EventCheckResult object
         """
-        stats = {}
-        all_dropped = []  # interfaces where all packets are dropped
-        all_errors = []  # interfaces where all packets are errors
-        ovs = OpenvSwitchBase()
```

Comment:
I see you're fixing some of the too-many-locals issues as well. I had done this as a class object (like event_group is done), but I guess as long as it's cached it's the same thing.